spec/dummy_controller_without_gems_spec.rb

```diff
@@ -150,4 +147,53 @@
       end
     end
   end
+
+  describe '#index_total_count' do
+    subject(:make_request) { get :index_total_count, params: { per: 5 } }
+
+    let!(:model_count) { 9 }
+    let!(:dummy_models) { create_list(:dummy_model, model_count) }
+    let(:expected_list) { dummy_models.first(5).as_json(only: %i[id name something]) }
+
+    before do
+      [Wor::Paginate::Adapters::Kaminari, Wor::Paginate::Adapters::WillPaginate].each do |klass|
+        allow_any_instance_of(klass).to receive(:adapt?).and_return(false)
+      end
+    end
+
+    context 'with total_count param' do
+      before { make_request }
+
+      include_examples 'total count pagination param'
+
+      it 'responds with valid page' do
+        expect(response_body(response)['page']).to eq expected_list
+      end
```

Comment:
You can move this example to shared examples `total count pagination params` and reuse it :smile: 
You are duplicating it in many specs :disappointed: